src/index.ts

```diff
@@ -51,8 +52,17 @@ class WebPhone extends EventEmitter {
               callSession.callId === inboundMessage.headers["Call-Id"],
           );
           if (index !== -1) {
-            this.callSessions[index].dispose();
+            const callSession = this.callSessions[index];
             this.callSessions.splice(index, 1);
+            callSession.dispose();
+          }
+        }
+
+        // NOTIFY message
+        if (inboundMessage.subject.startsWith("NOTIFY ")) {
+          const event = inboundMessage.headers.Event;
+          if (event === 'check-sync') {
+            this.emit("provisionUpdate");
```

Comment:
Instead of this, I would like to provide a general event "sip-message". So that you can get all messages to this call session and you can do whatever you want. 
This way, it is more flexible. And it is purely between SIP server and developer's code. The SDK doesn't do any pre-processing (avoid any magical and unnecessary code).

Reply:
Please use the following code:

```ts
webPhone.sipClient.on("inboundMessage", inboundMessage => {
  if(inboundMessage.headers.Event === "check-sync") {
    // do something as if you received the "provisionUpdate" event.
  }
});
```